docs/devel/README.md

```diff
@@ -0,0 +1,138 @@
+# Developer Setup
+
+This documents show how to build and run poseidon and other compenets on a dev setup.
+
+* Dependency 
+   * Kubernetes :- Running instance of a [kubernetes cluster](https://kubernetes.io/docs/setup/) is required. 
+   * Firmament  :- For firmament build info please refer [here](https://github.com/camsas/firmament/blob/master/README.md#building-instructions).

```

Comment:
Development work on Poseidon doesn't require building Firmament locally, does it? Presumably many people could just use a Docker image for testing if they're only changing Poseidon. Pointing this out may simplify peoples' lives a lot.